portal-api/src/v1/controllers/utilisation-report-service/utilisation-report-upload.controller.js

```diff
@@ -112,7 +113,6 @@ const uploadReportAndSendNotification = async (req, res) => {
 
     const existingReports = await api.getUtilisationReports(bankId, {
       reportPeriod: parsedReportPeriod,
-      excludeNotReceived: true,
```

Comment:
We should be checking for the existence of one report in the `REPORT_NOT_RECEIVED` state - `excludeNotReceived` will specifically remove reports in this state which we do not want